README.md

```diff
@@ -1 +1,70 @@
-# math-magicians
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
```

Comment:
- [ ] Kindly, Add a descriptive README file to your project A professional README file should have a project title, project description, list of technology used in the project, instructions to set the project up locally, .... please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.